Newsletters/Yearn Finance Newsletter #44/Yearn Finance Newsletter #44.md

```diff
@@ -1,95 +1,95 @@
+La votación reciente con respecto a YIP-63, ha resultado en el acuerdo de aportar fondos al DAO LeXpunK. Yearn ha transferido un millón de DAI al multifirma controlado por Curve Finance, Lido Finance y el DAO LeXpunK.
 
-For context, the LeXpunK DAO is a coalition of DeFi builders and lawyers with the goal of creating a legal defense team far greater and more ethically aligned than those at Silicon Valley venture funds. This is an important initiative that will serve to fight for crypto over the next few years or even decades.
+Como contexto, el DAO LeXpunK, es una coalición de constructores de Defi y abogados con el objetivo de crear un equipo de defensa legal mucho mayor y con una ética mas alineada que los VCs de silicon Valley. Es una iniciativa importante que servirá para luchar por el mundo cripto por los próximos años e incluso las próximas décadas.
 
-Read the full proposal on Yearn’s forum [here](https://gov.yearn.finance/t/yip-63-fund-builder-first-legal-activism-dao/11280).
+Lee la propuesta completa en el foro de Yearn [Aquí](https://gov.yearn.finance/t/yip-63-fund-builder-first-legal-activism-dao/11280).
 
-Check Yearn’s transaction to the DAO [here](https://etherscan.io/tx/0x0ec0fc55d6dc51b426a254bf2d6de138b1b9a1c3031f4ab3a7b39439fa004392)
+Echa un vistazo a la transacción realizada por Yearn para el DAO [Aquí](https://etherscan.io/tx/0x0ec0fc55d6dc51b426a254bf2d6de138b1b9a1c3031f4ab3a7b39439fa004392)
 
-## **Product User Survey**
+## **Encuesta a los usuarios**
 
-We have released a new user survey to gauge what you, the community at Yearn, would like to see added.  
-  
-Yearn is always iterating amongst ourselves and with the help of the community, but with this survey, we’d like to take a more hands on approach to seeing what you guys, the community, has in mind to improve Yearn. We have seen exponential growth in the short time we’ve been active, and we hope to continue innovating in the products we put out and add more utility to the YFI token. We think that we can improve the user experience at Yearn tenfold with the help of your thoughts, and thus become the most competitive in the yield aggregation space among the various other sectors we engage in.
+Hemos realizado una nueva encuesta de usuarios para determinar lo que vosotros, los usuarios de la comunidad, quereis ver en Yearn.
+ 
+Yearn se encuentra en constante proceso de iteración con la ayuda de la comunidad, pero con esta encuesta, queremos ponernos manos a la obra para así determinar lo que vosotros, la comunidad, tenéis en mente para mejorar Yearn. Hemos visto crecimiento exponencial en el corto tiempo que llevamos en actividad y esperamos continuar innovando en los productos que lanzamos al mercado y dando mayor utilidad al token YFI. Creemos que podemos mejorar la experiencia de usuario en Yearn con la ayuda de vuestras ideas y de esa forma ser mas competitivos en el mercado de generación de rendimiento así como en los distintos sectores donde nos desarrollamos.
```

Comment:
AS-IS: Yearn se encuentra en constante proceso de iteración con la ayuda de la comunidad, pero con esta encuesta, queremos ponernos manos a la obra para así determinar lo que vosotros, la comunidad, tenéis en mente para mejorar Yearn. Hemos visto crecimiento exponencial en el corto tiempo que llevamos en actividad y esperamos continuar innovando en los productos que lanzamos al mercado y dando mayor utilidad al token YFI. Creemos que podemos mejorar la experiencia de usuario en Yearn con la ayuda de vuestras ideas y de esa forma ser mas competitivos en el mercado de generación de rendimiento así como en los distintos sectores donde nos desarrollamos.

TO-BE: Yearn se encuentra en constante proceso de iteración con la ayuda de la comunidad, pero con esta encuesta, queremos ponernos manos a la obra para así determinar lo que vosotros, la comunidad, tenéis en mente para mejorar Yearn. Hemos visto crecimiento exponencial en el corto tiempo que llevamos en actividad y esperamos continuar innovando en los productos que lanzamos al mercado y dando mayor utilidad al token YFI. Creemos que podemos mejorar la experiencia de usuario en Yearn con la ayuda de vuestras ideas y de esa forma ser más competitivos en el mercado de generación de rendimiento así como en los distintos sectores donde nos desarrollamos.